src/pudl/transform/params/ferc1.py

```diff
@@ -4372,6 +4415,245 @@
             },
         ],
     },
+    "electricity_sales_by_rate_schedule_ferc1": {
+        "rename_columns_ferc1": {
+            "dbf": {
+                "columns": {
+                    "respondent_id": "utility_id_ferc1_dbf",
+                    "report_year": "report_year",
+                    "report_prd": "report_prd",
+                    "row_prvlg": "row_prvlg",
+                    "row_number": "row_number",
+                    "row_seq": "row_seq",
+                    "spplmnt_num": "spplmnt_num",
+                    "sched_num_ttl": "description_of_number_and_title_of_rate_schedule",
+                    "mwh_sold": "sales_mwh",
+                    "revenue": "sales_revenue",
+                    "avg_num_cstmr": "avg_customers_per_month",
+                    "kwh_sale_cstmr": "kwh_per_customer",
+                    "revenue_kwh_sold": "revenue_per_kwh",
+                }
+            },
+            "xbrl": {
+                "columns": {
+                    "entity_id": "utility_id_ferc1_xbrl",
+                    "report_year": "report_year",
+                    "sales_axis": "sales_axis",
+                    "average_number_of_customers_per_month_residential_sales": "residential_total_avg_customers_per_month",
```

Comment:
non-blocking, idle musing... it'd be nice if we could encode the remapping with regex match groups and such...

something like
```(.+)(_residential_sales|_small_or_commercial|_per_month_sales_of_electricity_by_rate_schedules)(_unbilled|_billed)?```

Would let us pull the relevant things out of match groups and rename programmatically - here's what the matches look like on regex101:
![image](https://user-images.githubusercontent.com/2495794/217357836-9a5d770b-e0a7-494f-ab76-17775013b753.png)

Reply:
This would be nice, but I don't want to put any effort into reworking the `rename_columns` function atm